container/concurrent/lrucache.go

```diff
@@ -0,0 +1,89 @@
+package concurrent
+
+import (
+	"sync"
+	"github.com/dropbox/godropbox/container/lrucache"
+)
+
+type LRUCache interface {
+	GetMultiple(keys []string) []CacheResult
+	Get(key string) (v interface{}, found bool)
+	Set(key string, v interface{})
+	SetMultiple(keyValues map[string]interface{})
+	Delete(keys ...string)
+	Clear()
+}
+
+type CacheResult struct {
+	V     interface{}
+	Found bool
+}
+
+type concurrentLruCacheImp struct {
+	cache *lrucache.LRUCache
+	lock  sync.RWMutex
+	size  int
```

Comment:
not need to store this here.  You should expose `MaxSize() int` as part of the LRUCache interface, and expose the MaxSize from the underlying cache.